jdaviz/app.py

```diff
@@ -272,7 +272,11 @@ def load_data(self, file_obj, parser_reference=None, **kwargs):
             # Properly form path and check if a valid file
             file_obj = pathlib.Path(file_obj)
             if not file_obj.exists():
-                raise FileNotFoundError("Could not locate file: " + file_obj)
+                msg_text = "Error: File {} does not exist".format(file_obj)
+                snackbar_message = SnackbarMessage(msg_text, sender=self,
+                                                   color='error')
+                self.hub.broadcast(snackbar_message)
+                return
```

Comment:
```suggestion
                raise FileNotFoundError("Could not locate file: " + file_obj)
```

Reply:
Interestingly, it turns out that the problem was that the error was not being raised when using `+` for the string concatenation inside the error call. Changing it to the way I normally use, `"Could not locate file: {}".format(file_obj)` made the error get raised as expected. I left in the snackbar message as well, but now it should do both.

Reply:
Oh, I understand now - file_obj is a Path not a string, and you can't use `+` to concatenate `PosixPath` and `str`.